project/frontend/src/components/ResultsMap.js

```diff
@@ -15,19 +15,27 @@ function MapContainer(props) {
 
   // TODO: Add marker for "you are here location".
   const MarkerIcon = (props) => {
-    const markerName = 'marker' + props.id;
+    const markerID = props.id;
+    const markerName = `marker${props.id}`;
```

Comment:
Didn't realize you were also using variable `markerID` below. Since you are you can use it here as well instead of props.id